sdks/java/core/src/test/java/org/apache/beam/SdkCoreApiSurfaceTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam;
+
+import com.google.common.collect.ImmutableSet;
+import java.io.IOException;
+import java.util.Set;
+import org.apache.beam.sdk.util.ApiSurface;
+import org.apache.beam.sdk.util.ApiSurfaceVerificationTest;
+import org.hamcrest.Matcher;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * API surface verification for {@link org.apache.beam}.
+ */
+@RunWith(JUnit4.class)
+public class SdkCoreApiSurfaceTest extends ApiSurfaceVerificationTest {
+
+  @Override
+  protected ApiSurface apiSurface() throws IOException {
+    return ApiSurface.getSdkApiSurface();
+  }
+
+  @Override
+  protected Set<Matcher<Class<?>>> allowedClasses() {
+    //noinspection unchecked
+    return ImmutableSet.of(inPackage("org.apache.beam"),
```

Comment:
Given that this `Set` just represents `or`, I think you could just make this one `Matcher`.